README.md

```diff
@@ -346,7 +366,7 @@ Our Neural Speed supports  INT4 / INT8 / FP8 (E4M3, E5M2) / FP4 (E2M1) / NF4 wei
 | NF4 | BF16 / FP16 / FP32 (FP32) | BF16 / FP32 (FP32) | sym (sym) |
 
 
-### 2. Inference LLM
+### 2. Inference
 
 We provide LLM inference script to run the quantized model. Please reach [us](mailto:itrex.maintainers@intel.com) if you want to run using C++ API directly.
```

Comment:
add enabling new models from https://github.com/intel/intel-extension-for-transformers/blob/main/intel_extension_for_transformers/llm/runtime/graph/developer_document.md

Reply:
how about where it already is? https://github.com/intel/neural-speed/blob/a262c61ec691f238f3de45c1dc172bf2ceeadc96/README.md#4-contribution

Reply:
looks good, but please simple descriptions